tests/manage/mcg/test_virtual_hosted_buckets.py

```diff
@@ -0,0 +1,94 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.bucket_utils import (
+    verify_s3_object_integrity,
+    write_random_objects_in_pod,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    tier2,
+    polarion_id,
+    red_squad,
```

Comment:
@mashetty330 could you please add mark(rgw or mcg) as part of our Test Directory Restructure

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/9070 
Above PR is not merged yet so we can ignore mark changes for now

Reply:
@udaysk23 we can take care of it in the open PR https://github.com/red-hat-storage/ocs-ci/pull/9070